### PR TITLE
Delete submission api

### PR DESCRIPTION
**Pull request by LeBat**

### CODE REVIEW

File: src/db.py [hunk 1]
```diff
@@ -239,6 +239,14 @@ def create_submission(self, url_name, long_name, parent_project_url, owner):
 
         return submission_db.insert_one(submission_obj)
 
+    def delete_submission(self, long_name, owner, gitolite_url):
+        submission_db = self.mongo.gitsubmit.submissions
+        submission_obj = submission_db.find({"long_name": long_name, "owner": owner, "gitolite_url": gitolite_url})
+        
+        if submission_obj is not None:
+            submission_db.drop(submission_db)
```
Comment (shawkinsl) [inline src/db.py]: this line would delete the entire database. I'm pretty sure you meant `submission_db.remove(submission_obj["_id"])`

Comment (shawkinsl) [inline src/db.py]: I will fix this and merge.

**Comment by shawkinsl** [comment]
Also, in the future please try to keep any working branches up to date with master. The tests have changed significantly for this project.

**Comment by LeBat** [comment]
Will remember that in the future. Sorry about that.

**Pull request merged**